### DISCUSSION TITLE
Jarvis Konfiguration verschwunden nach Stromausfall

### DISCUSSION THREAD

**Discussion by msiemers** [Zefau/ioBroker.jarvis#759] [Q&A]
Meine Jarvis Installation begrüßte mich heute nach einem Stromausfall mit der Aufforderung eine neue Konfiguration zu erstellen. Kann ich hier noch etwas machen um meine alten Einstellungen wieder zu bekommen?

**Comment by Zefau** [comment]
jarvis hat eine restore Funktion in den ioBroker Adaptereinstellungen. Ansonsten via backitup wiederherstellen.

**Comment by msiemers** [comment]
Die Restore Funktion ist leer. Und den ganzen Iobroker zurücksetzen möchte ich nicht.
Am 9. Mai 2021, 00:20 +0200 schrieb Zefau ***@***.***>:

**Comment by Zefau** [comment]
Der backitup Adapter müsste ein Archiv erstellt (tar.gz?). Darin findest du jarvis und die JSON files. Frag mal im Forum nach.

**Comment by msiemers** [comment]
@Zefau in welchem Forum?

**Comment by Zefau** [comment]
https://forum.iobroker.net/topic/30668/jarvis-v2-2-0-just-another-remarkable-vis/
